roxie/ccd/ccdqueue.cpp

```diff
@@ -1374,8 +1374,12 @@ class CRoxieWorker : public CInterface, implements IPooledThread
                 CriticalBlock b(actCrit);
                 EXCLOG(E);
                 if (packet)
+                {
                     throwRemoteException(E, NULL, packet, false);
-                packet.clear();
+                    packet.clear();
```

Comment:
slightly redundant/confusing since throw should never return

Reply:
I take it back - it looks like throwRemoteException doesn't actually throw an exception.  Possibly scope for renaming!

Reply:
It's throwing it to the server... I agree could be renamed (though not a new issue)